### PR TITLE
Make interative mode able to set to false

### PR DESCRIPTION
**Pull request by houshengbo**
The default value of interative mode should be false, instead of depending
on utils.Flags.WithinOpenWhisk.

Closes-Bug: #374

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -113,7 +113,7 @@ func init() {
 	RootCmd.Flags().StringVarP(&utils.Flags.ProjectPath, "pathpath", "p", ".", "path to serverless project")
 	RootCmd.Flags().StringVarP(&utils.Flags.ManifestPath, "manifest", "m", "", "path to manifest file")
 	RootCmd.Flags().StringVarP(&utils.Flags.DeploymentPath, "deployment", "d", "", "path to deployment file")
-	RootCmd.PersistentFlags().BoolVarP(&utils.Flags.UseInteractive, "allow-interactive", "i", !utils.Flags.WithinOpenWhisk, "allow interactive prompts")
```
Comment (daisy-ycguo) [inline cmd/root.go]: I agree with this change.

File: deployers/servicedeployer.go [hunk 2]
```diff
@@ -119,7 +119,7 @@ func (deployer *ServiceDeployer) ConstructDeploymentPlan() error {
 
 	manifestReader.InitRootPackage(manifestParser, manifest)
 
-	if deployer.IsDefault == true && !utils.Flags.WithinOpenWhisk {
+	if deployer.IsDefault == true {
```
Comment (daisy-ycguo) [inline deployers/servicedeployer.go]: I hope to include David to review this line of change.

File: deployers/servicedeployer.go [hunk 3]
```diff
@@ -183,7 +183,7 @@ func (deployer *ServiceDeployer) ConstructUnDeploymentPlan() (*DeploymentApplica
 // TODO(TBD): according to some planning?
 func (deployer *ServiceDeployer) Deploy() error {
 
-	if deployer.IsInteractive == true && !utils.Flags.WithinOpenWhisk {
+	if deployer.IsInteractive == true {
```
Comment (daisy-ycguo) [inline deployers/servicedeployer.go]: I hope to include David to review this line of change.

**Comment by pritidesai** [comment]
LGTM,  thanks!

File: deployers/servicedeployer.go (hunk 2 above)
Comment (lzbj) [inline deployers/servicedeployer.go]: thanks @daisy-ycguo , LGTM.

File: deployers/servicedeployer.go [hunk 4]
```diff
@@ -119,7 +119,7 @@ func (deployer *ServiceDeployer) ConstructDeploymentPlan() error {
 
 	manifestReader.InitRootPackage(manifestParser, manifest)
 
-	if deployer.IsDefault == true && !utils.Flags.WithinOpenWhisk {
```
Comment (mrutkows) [inline deployers/servicedeployer.go]: I am curious as why we were testing "WithinOpenWhisk" here in the first place...

Comment (pritidesai) [inline deployers/servicedeployer.go]: @lionelvillard had added this flag to indicate that the wskdeploy is being run as an OpenWhisk action itself. If it is being run as an OpenWhisk action than skip this whole block of execution where wskdeploy is reading and inspecting project directory where action files are located, reading content of the action files and setting exec.Code of an ActionRecord. Hope this helps. We cannot set exec.Code this way if wskdeploy is running as an OpenWhisk action as none of the directory structure is accessible in cloud. But than I am trying to figure out myself what is the use of having wskdeploy run as an OpenWhisk action.

Comment (davidbreitgand) [reply deployers/servicedeployer.go]: With the new "export" capability added by @kpavel, the directory structure will be exported to the FS of the container, in which the action runs. The use case would be: "export" assets from one OW instance and "import" into another one via running wskdeploy from within an action.

Comment (davidbreitgand) [reply deployers/servicedeployer.go]: With the new "export" capability added by @kpavel, the directory structure will be exported to the FS of the container, in which the action runs. The use case would be: "export" assets from one OW instance and "import" into another one via running wskdeploy from within an action.

**Pull request merged**